app/pages/project/classify.cjsx

```diff
@@ -92,6 +94,16 @@ module.exports = React.createClass
   componentWillUnmount: () ->
     @context.geordi?.forget ['workflowID']
 
+  componentWillReceiveProps: (nextProps) ->
+    @shouldWorkflowAssignmentPrompt(nextProps.preferences)
+
+  shouldWorkflowAssignmentPrompt: (preferences) ->
+    # Only for Gravity Spy which is assigning workflows to logged in users
+    assignedWorkflowID = preferences?.settings?.workflow_id
+    if @props.project.experimental_tools.indexOf 'workflow assignment' > -1 and @props.user?
```

Comment:
Probably would be a good idea to include the parentheses anyway. Thanks for catching that again here.